contrib/normalize_triplet.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
```

Comment:
Why is this in Python?

Reply:
Because I didn't want to rewrite [this](https://github.com/JuliaPackaging/BinaryProvider.jl/blob/master/src/PlatformNames.jl#L365-L452) in `awk`, and we already require `python` to build LLVM, so it's not adding a new build dependency.  I like the `python` implementation because it is still close enough to the source Julia that co-maintaining them shouldn't be too big of a problem.

Reply:
Could we not use Julia here?

Reply:
Well that would require a `julia` while we're trying to build `julia`.  Perhaps someday, but I don't think we can assume that everybody has a copy of `julia` yet.....

Reply:
done